python/cudf/cudf/utils/applyutils.py

```diff
@@ -143,17 +143,20 @@ def run(self, df, **launch_params):
         # Get input columns
         if isinstance(self.incols, dict):
             inputs = {
-                v: df[k]._column.data_array_view
+                v: df[k]._column.data_array_view(mode="write")
                 for (k, v) in self.incols.items()
             }
         else:
-            inputs = {k: df[k]._column.data_array_view for k in self.incols}
+            inputs = {
+                k: df[k]._column.data_array_view(mode="write")
+                for k in self.incols
+            }
```

Comment:
Given these are inputs to a kernel launch that don't escape this function, can they not be `read` mode accesses?

Reply:
You are correct, not needed here too.

Reply:
Fixed it.